### PR TITLE
Bug 1238533 - Clicker UI notification UI

### PR DESCRIPTION
**Pull request by mancas**

### CODE REVIEW

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: standalone/content/js/standaloneRoomViews.jsx [hunk 1]
```diff
@@ -522,6 +597,73 @@ loop.standaloneRoomViews = (function(mozL10n) {
     }
   });
 
+  var StandaloneNotificationView = React.createClass({
```
Comment (daveccrit) [inline standalone/content/js/standaloneRoomViews.jsx]: ESLint is saying StandaloneNotificationView is defined and never used.   Likely just needs to be added to the return statement on line 971 to fix.

File: standalone/content/js/standaloneRoomViews.jsx [hunk 2]
```diff
@@ -522,6 +597,73 @@ loop.standaloneRoomViews = (function(mozL10n) {
...
+            </div>
+          </div>
+        </div>
+      );
+    }
+  });
+
   var StandaloneRoomView = React.createClass({
     mixins: [
       Backbone.Events,
```
Comment (daveccrit) [inline standalone/content/js/standaloneRoomViews.jsx:590]: There are hardcoded strings and no unit tests, was everything pushed in the commit?

**Pull request merged**